### ISSUE TITLE
Another data discrepancy?

### ISSUE THREAD

**Issue by djhunter** [UCSBCarpentry/CustomDC-R#6]
The last example on the Counting section in Episode 4 says this:

> From the table above, we may learn that, for instance, there are 75 observations of the albigula species that are not specified for its sex (i.e. NA).

However, the only species are Adelie Chinstrap Gentoo. Was this written using a different data set? 

Here's what the code in the lesson does:

```r
penguins %>%
  count(sex, species) %>%
  arrange(species, desc(n))
```

Output:
```
# A tibble: 8 × 3
  sex    species       n
  <fct>  <fct>     <int>
1 female Adelie       73
2 male   Adelie       73
3 NA     Adelie        6
4 female Chinstrap    34
5 male   Chinstrap    34
6 male   Gentoo       61
7 female Gentoo       58
8 NA     Gentoo        5
```

Should it say the following?

> From the table above, we may learn that, for instance, there are 6 observations of the Adelie species that are not specified for its sex (i.e. NA).

**Comment by apandas** [comment]
this was another case of data discrepancy, wording has been corrected in commit [78dbf88](https://github.com/UCSBCarpentry/CustomDC-R/commit/78dbf8868c9b851b66fd3a32140f01f8b8c8ab6f)

**Comment by apandas** [comment]
Mistyped issue #6, commit [5f434fd](https://github.com/UCSBCarpentry/CustomDC-R/commit/5f434fd2d40f5a051f395b37df9b48439b108ef2) does not address this issue
